src/components/auction/OrderPlacement/index.tsx

```diff
@@ -219,12 +219,18 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
     setAttemptingTxn(false)
   }
 
+  const auctioningToken = React.useMemo(() => derivedAuctionInfo?.auctioningToken, [
```

Comment:
If we move these defintions up several lines, we can reuse this definition even more times.